exercises/simple-cipher/.meta/version

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
We might as well bump this to version 2.0.0 are remove the step three tests so that the tests are fully up to date? 🙂

Reply:
Done. 👍